pkg/cmd/preview/preview.go

```diff
@@ -1001,18 +1001,12 @@ func (o *PreviewOptions) getContainerRegistry(projectConfig *config.ProjectConfi
 
 func (o *PreviewOptions) getImageName(projectConfig *config.ProjectConfig) (string, error) {
 	containerRegistry, err := o.getContainerRegistry(projectConfig)
+	dockerRegistryOrg := o.GetDockerRegistryOrg(projectConfig, o.GitInfo)
+
 	if err != nil {
 		return "", err
 	}
 
-	organisation := os.Getenv(ORG)
-	if organisation == "" {
-		organisation = os.Getenv(REPO_OWNER)
```

Comment:
Huh, I was nervous at first looking at this, but now I see that this was all just fallbacks for setting `dockerRegistryOrg`, so this was just silly code in the first place. This feels like a real improvement!

Reply:
thanks :)